### DISCUSSION TITLE
how to enable a scroll function

### DISCUSSION THREAD

**Discussion by wukimidaire** [spider-rs/spider#235] [Q&A] [answered]
hi there superstars,

I'm looking into how to enable a scroll function into the spider.cloud api.
Currently using a curl request and n8n low-code software setup.

Could not find anything on how to use scroll into my requests yet so directing my question here.

Thanks in advance for any information related to this.

**Comment by j-mendez** [comment]
> hi there superstars,
> 
> 
> 
> I'm looking into how to enable a scroll function into the spider.cloud api.
> 
> Currently using a curl request and n8n low-code software setup.
> 
> 
> 
> Could not find anything on how to use scroll into my requests yet so directing my question here.
> 
> 
> 
> Thanks in advance for any information related to this.
> 
> 
> 
> 

Hi there! We have a param called "scroll" for infinite scrolling that you put a duration of time up to.

**Reply by wukimidaire** [reply]
can you please point me in the direct section/direction as I can't find it in any documentation on the web interface right now :s

**Reply by j-mendez** [reply, marked as answer]
> can you please point me in the direct section/direction as I can't find it in any documentation on the web interface right now :s 

It is under the automation tab the first param on the list on the API docs.

**Reply by wukimidaire** [reply]
Found the settings on the api documentation. Thank you!
